### PR TITLE
Meeting sidebar styles

### PR DESCRIPTION
**Pull request by ackernaut**
#### In the meeting navigation sidebar:
- [ ] The header has a meeting name and a link to the team dashboard
- [ ] The assign facilitator control is styled
- [ ] The meeting phase items have icons instead of ordered bullets
- [ ] It highlights what phase the facilitator is in when the local user is not in the same phase
- [ ] It highlights what stage the facilitator is in when the local user is not in the same stage
- [ ] Stage items (Team Agenda, Discuss) don’t have ordered bullets
- [ ] The agenda input uses the blue color for cool emphasis opposed to warm emphasis
- [ ] Stage items for team member rounds (Social Check-In, Solo Updates) to come in #3238 

#### Behaviors:
- [ ] Agenda sidebar works on team dashboard
- [ ] Can collapse/open navigation sidebar for action meeting
- [ ] Meeting navigation sidebar works for the action meeting lobby
- [ ] Meeting navigation sidebar works for the active action meeting
- [ ] Can drag and drop agenda items
- [ ] Action meeting summary works
- [ ] Can collapse/open navigation sidebar for retro meeting
- [ ] Meeting navigation sidebar works for the retro meeting lobby
- [ ] Meeting navigation sidebar works for the active retro meeting
- [ ] Can drag and drop discuss topics
- [ ] Retro meeting summary works

### Note
Merge this before merging #3238

### CODE REVIEW

**Comment by ackernaut** [comment]
@mattkrick this is ready for a review. A few things:
- As facilitator, when you advance to the Discuss phase the first time, you see the Rejoin Facilitator button flash for a brief moment. This wasn’t introduced on this branch; I was able to reproduce on staging. I just noticed now since I was looking closely at those styles.
- Once the Discuss phase has been reached, I would like that section of stage nav items to stay open when you navigate back to a previous phase. I tracked it down to L75 of `RetroSidebarDiscussSection` but didn’t figure it out yet. Not a deal-breaker for me. Just had me stumped for a minute.
- I made some progress on stage nav items for Social Check-In and Solo Updates on another PR, but wanted to get these styles out since the assign facilitator control merged.
Let me know what you think about the UX. You won’t see a facilitator color on phase or stage unless you break sync with another user. I’m okay with that being blue but I’d want to go back and change other places first. I can do that on the next branch.

**Comment by mattkrick** [comment]
> Once the Discuss phase has been reached, I would like that section of stage nav items to stay open when you navigate back to a previous phase.

```
const [voteStage] = phases.find((phase) => phase.type === NewMeetingPhaseTypeEnum.vote)
const isAlwaysOpen = voteStage.isComplete
```

nit: the expanded hamburger is no longer in the same place as the collapsed hamburger.  

also, i don't see discuss items in the sidebar. is there another commit somewhere to fix this?

note: icons look freaking AWESOME!

**Comment by ackernaut** [comment]
@mattkrick this is ready…I added some behaviors to the tests just to think through everything that should work. Going to test on `development` myself.

**Comment by mattkrick** [comment]
@ackernaut looks like there's still a couple typescript errors, mostly lil stuff. check out the Typecheck / build to see em

**Comment by mattkrick** [comment]
### Tests
#### In the meeting navigation sidebar:
- [ ] The header has a meeting name and a link to the team dashboard
- [ ] The assign facilitator control is styled
- [ ] The meeting phase items have icons instead of ordered bullets
- [ ] It highlights what phase the facilitator is in when the local user is not in the same phase
- [ ] It highlights what stage the facilitator is in when the local user is not in the same stage
- [ ] Stage items (Team Agenda, Discuss) don’t have ordered bullets
- [ ] The agenda input uses the blue color for cool emphasis opposed to warm emphasis
- [ ] Stage items for team member rounds (Social Check-In, Solo Updates) to come in #3238 

#### Behaviors:
- [ ] Agenda sidebar works on team dashboard
- [ ] Can collapse/open navigation sidebar for action meeting
- [ ] Meeting navigation sidebar works for the action meeting lobby
- [ ] Meeting navigation sidebar works for the active action meeting
- [ ] Can drag and drop agenda items
- [ ] Action meeting summary works
- [ ] Can collapse/open navigation sidebar for retro meeting
- [ ] Meeting navigation sidebar works for the retro meeting lobby
- [ ] Meeting navigation sidebar works for the active retro meeting
- [ ] Can drag and drop discuss topics
- [ ] Retro meeting summary works

File: packages/client/components/ActionMeetingSidebar.tsx [hunk 1]
```diff
@@ -51,22 +63,21 @@ const ActionMeetingSidebar = (props: Props) => {
...
                     : localPhaseType === phaseType
                 }
-                isFacilitatorPhaseGroup={
-                  facilitatorPhaseType === phaseType ||
-                  (phaseType === NewMeetingPhaseTypeEnum.agendaitems &&
-                    blackList.includes(facilitatorPhaseType))
-                }
+                isUnsyncedFacilitatorPhase={isUnsyncedFacilitatorPhase && phaseType === facilitatorPhaseType}
                 handleClick={canNavigate ? handleClick : undefined}
+                meta={phaseCount}
```
Comment (mattkrick) [inline packages/client/components/ActionMeetingSidebar.tsx]: `meta` could be a little more expressive.
maybe call it `stageCount` have it take `undefined | null` to be hidden or a number to show?

**Comment by mattkrick** [comment]
when i hover the facilitator dropdown, it shows an outline. this feels a little dated & it's the only place in the whole app where we add a border on hover. could we instead just add a gray circle background to the down arrow or something more subtle?

File: packages/client/components/AssignFacilitator.tsx [hunk 2]
```diff
@@ -24,7 +26,7 @@ const AssignFacilitatorToggle = styled('div')<{isActive: boolean, isReadOnly: bo
   display: 'flex',
   color: isActive ? PALETTE.TEXT_MAIN : PALETTE.TEXT_GRAY,
   cursor: isReadOnly ? undefined : 'pointer',
-  padding: '4px 8px',
+  padding: '4px 7px 4px 4px',
```
[the remark this answers is not in the document]
Comment (ackernaut) [reply packages/client/components/AssignFacilitator.tsx]: Yes, I was compensating for the border of the control that appears on hover. I’ll reconsider even using a border. But the overall alignment from top to bottom of the sidebar is on the 8px grid. This is especially key for how icons and left edges of text align across different components (header, controls, nav items). So when there is any hanging of elements, then there is compensation. I’ll double check after considering the border vs. hover styling.

File: packages/client/components/ActionMeetingSidebar.tsx (hunk 1 above)
Comment (ackernaut) [reply packages/client/components/ActionMeetingSidebar.tsx]: Sounds good.

**Comment by ackernaut** [comment]
@mattkrick I like the idea of something more subtle. I didn’t like how the border was eating into the 8px grid. Also, I’m hanging the connection circle outside of the 8px grid so that the avatar size is 24px. So there was some compensation there, too.

**Comment by ackernaut** [comment]
>feels weird that the draggable has such a long left padding. we should pad the children before the draggables so the think they're dragging doesn't have like 48px of emptiness on the left side.

@mattkrick I noticed that too, but I think it also feels weird for the item to shrink then be wide again, plus to do that calls for some funky styling hacks IMO. I’m gonna pass on that detail for now, but I do understand the nuance.

**Comment by mattkrick** [comment]
After looking at the facilitator changer through fresh eyes... do we even need a dropdown?
The pattern of promoting someone else to facilitator when I'm _not_ the facilitator is a little strange.
Heck, it might even be strange when I am the facilitator.
Instead of the down arrow, can we just have a "make me the facilitator" button? that'll eliminate a menu & keep folks from foot gunning themselves like User B promotes User C to facilitator without their knowledge

**Pull request merged**